test/emoji-util.test.js

```diff
@@ -0,0 +1,64 @@
+const emojiUtil = require('../src-electron/util/emoji-util')
+const env = require('../src-electron/util/env')
+
+describe('Emoji Utility Tests', () => {
```

Comment:
Will be better to write this test based on the function uc_generate calls to actually resemble the real use case and test what Studio will be doing